test/check-application

```diff
@@ -1913,6 +1913,36 @@ class TestApplication(testlib.MachineCase):
         self.performContainerAction(container_name, "Resume")
         b.wait(lambda: self.getContainerAttr(container_name, "State") == "Running")
 
+    def testRenameContainerSystem(self):
+        self._testRenameContainer(True)
+
+    def testRenameContainerUser(self):
+        self._testRenameContainer(False)
+
+    def _testRenameContainer(self, auth):
+        b = self.browser
+        container_name = "rename"
+        container_name_new = "rename-new"
+        
+        self.execute(auth, f"podman container create -t --name {container_name} quay.io/libpod/busybox:latest")
+        self.login(auth)
+
+        self.filter_containers('all')
+
+        self.waitContainerRow(container_name)
+        self.toggleExpandedContainer(container_name)
+        self.performContainerAction(container_name, "Rename")
+
+        # the container name should be in the "Rename container" header
+        self.assertIn(container_name, b.text("#pf-modal-part-1"))
+
+        b.set_input_text("#rename-dialog-container-name", container_name_new)
+        b.click('#btn-rename-dialog-container')
+        b.wait_not_present("#rename-dialog-container-name")
+
+        container_sha = self.execute(auth, f"podman inspect --format '{{{{.Id}}}}' {container_name_new}").strip()
+        self.waitContainer(container_sha, auth, name=container_name_new, image='busybox:latest', cmd='sh',
+                           state='Configured', owner="system" if auth else "admin")
```

Comment:
Well done on the test! Can you please now start the container, and pin down the behaviour for running containers? Judging by the code, the intention is that there should not be a "Rename" action, so the test should ensure that the kebab does not have that menu entry any more. Then, if at some point running container names are supported/get implemented, we have a reminder to update this.

Reply:
Added.

Reply:
>     * When a string doesn't match the regular expression, it should say something about invalid characters... or it should just filter the characters like the Machines page does for naming. (Where invalid ones are stripped and spaces are changed to dashes or underscores, whichever is supported, unless spaces are simply supported in the names... then they shouldn't be stripped.) It should _not_ be an alert.

I added a regexp validator for name, have a look. Machines names can now have non-ASCII characters, unlike Containers.